domains/wazin.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "WazinAjmal",
+    "email": "wazinajmal@gmail.com",
```

Comment:
Invalid JSON. You placed a comma in a place that shouldn't have one. Double check JSON and re-open when fixed and checks pass.